recipes/pyston_lite/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set name = "pyston_lite" %}
+{% set pyston_version = "2.3.4" %}
+{% set python_version = "3.8.* *_cpython" %}
+{% set build_num = 0 %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ pyston_version }}
+
+source:
+  - url: https://github.com/kmod/pyston/archive/bcbb8e939c28162e39d536622625961522424117.tar.gz
+    sha256: 753f3687d49173e860c737758b81d78419b74f3fd05abe6f05eb06c072cbf42c
+  # pyston-lite uses a preprocessing script from inside the LuaJIT repository (that is not
+  # packaged as part of the luajit package), but only as a build tool (ie is not included
+  # in the built artifacts).
+  - url: https://github.com/LuaJIT/LuaJIT/archive/e2c312e0deb874aa5fa8ce502c08d87deb38e82f.tar.gz
+    sha256: bff4764c19466def90667d2f7fcbd0781b8e62476b12105868f8e44babd03f8e
+    folder: pyston/LuaJIT/
+
+build:
+  number: {{ build_num }}
```

Please put a plain number here, it will be propagated to the outputs
```suggestion
  number: 0
```